src/shape-editors/polygon-editor.js

```diff
@@ -25,6 +25,46 @@ export default class PolygonEditor extends BaseShapeEditor {
         this._el.polygon.deactivate();
     }
 
+
+    onKeyPressed(key, altKey, shiftKey) {
+        let pointMoveGap = 0.01;
```

Comment:
I think it is better to rely on pixels (maybe considering devicePixelRation too), because 0.01 on 10x10px and 1000x1000px images are 100 times different